src/cli/paths.rs

```diff
@@ -35,18 +32,26 @@ pub mod non_wasm {
     }
 
     pub fn repl_history() -> PathBuf {
-        lurk_dir().join(Path::new("repl-history"))
+        lurk_dir().join("repl-history")
     }
 
     pub fn commitment_path(name: &str) -> PathBuf {
-        commits_dir().join(Path::new(name))
+        commits_dir().join(name)
     }
 
     pub fn proof_path(name: &str) -> PathBuf {
-        proofs_dir().join(Path::new(name)).with_extension("proof")
+        proofs_dir().join(name).with_extension("proof")
     }
 
     pub fn proof_meta_path(name: &str) -> PathBuf {
-        proofs_dir().join(Path::new(name)).with_extension("meta")
+        proofs_dir().join(name).with_extension("meta")
+    }
+
+    pub fn circom_gadgets() -> PathBuf {
+        lurk_dir().join("circom-gadgets")
+    }
+
+    pub fn circom_binary() -> PathBuf {
+        circom_gadgets().join("circom")
```

Comment:
what's the idea behind adding the gadgets prefix?

Reply:
I wanted to organize all circom related things under the `circom-gadgets` directory. Now that I type it out though, maybe `circom-gadgets` should be renamed to `circom`